server/controllers/users/profile.js

```diff
@@ -50,6 +51,17 @@ export const update = async function(req, res) {
   if (sameEmail && sameEmail._id !== req.user._id)
     throw new BadRequestError('Email address is taken')
 
+  // Update admin status
```

Comment:
Lines 54-62 should only be ran if `user` has the `ADMIN_ROLE`.

Reply:
Right. I'll need to update `<EditUser>` as well. Forgot all of this was being used for two different purposes.

Reply:
There is already a check for admin in the `UserRouter.js` file that loads <EditUser>.  Currently this component is only being used for admins to edit user accounts.  So I think it would be OK to just do the check on the server side for now.